websocketOutportDriver/sender/acknowledgesHolder.go

```diff
@@ -0,0 +1,38 @@
+package sender
+
+import "sync"
+
+type acknowledgesHolder struct {
```

Comment:
No need to do anything here, just a remark.
The way this is implemented now (by returning the pointers on Get), I think you can "compress" `acknowledgesHolder` and `websockerAcnowledgesHolder`.

Reply:
it all started from avoiding having a `map[string]map[uint64]struct{}` so I would let it as it is now